src/queries/useTrip.ts

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { fetchTripHandler } from '@/functions/fetchTripHandler';
+import { LocationProps } from '@/types/Types';
+
+export function useTrip(restructureTripData: LocationProps[]) {
+  return useQuery({
+    queryKey: ['valhalla', restructureTripData],
```

Comment:
If you start working with entities I would recommend organizing the query keys: https://slides.kilianfinger.com/fhs-2024-ss-mmt-m-frontend-engineering/57

But for those queries which never get invalidated I don't request you to change it. Just keep in mind for maybe queries which will be added onwards.

Reply:
I'm not sure here. we have two queries, which query could get invalidated from your point of view?

Reply:
Was only a reminder for later. If you add a backend where the user profile with routes are saved you will start invalidating queries. Then it's helpful to organize the query keys.